ga4gh/backend.py

```diff
@@ -292,6 +292,12 @@ def setResponseValidation(self, responseValidation):
         self._responseValidation = responseValidation
 
 
+class EmptyBackend(AbstractBackend):
```

Comment:
I thought it would be useful for testing - there are times when we want to assert some properties of the backend when we don't care about data at all (`test_views` uses it now I think). I didn't want to instantiate `AbstractBackend` directly as this feels a bit wrong, and we might change the behaviour of `AbstractBackend` later to now be directly instantiable.